### PR TITLE
`frolint` Version Update `3.0.0` to `3.1.1`

### PR DESCRIPTION
**Pull request by s-kawabe**
## Why

https://github.com/wantedly/frolint/pull/1102

## What

- update`frolint` version 
- update CHANGELOG.md

## TODO

- Run npm publish locally

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "packages": ["packages/*"],
-  "version": "3.0.1",
```
Comment (s-kawabe) [inline lerna.json:3]: 🗒 `3.0.1` change was not published.

**Comment by s-kawabe** [comment]
@chloe463 
please review 🙏🏻

**Review by chloe463 — APPROVED** [review]
LGTM

**Comment by s-kawabe** [comment]
cc: @chloe463 

- [x] yarn test
- [x] yarn build 
- [x] cd packages/frolint
- [x] npm publish
- [x] Published 3.1.1 🎉 https://www.npmjs.com/package/frolint

**Pull request merged**